src/plugins/js-eval/run.js

```diff
@@ -10,6 +10,8 @@ const inspect = (val) => {
       maxArrayLength: 20,
       breakLength: Infinity,
       colors: false,
+      compact: 10,
+      depth: 10,
```

Comment:
This will make output less verbose, currently we can see sometimes: `[ 1, 2, 3,   4, 5, 6,   7, 8, 9 ]` due to line returns